.github/workflows/sync_from_airtable.yml

```diff
@@ -0,0 +1,330 @@
+name: Sync from Airtable
+
+on:
+  repository_dispatch:
+    types: [airtable-update]
+
+env:
+  DATA: ${{ toJSON(github.event.client_payload.data) }}
+  NUMBER: ${{ github.event.client_payload.number }}
+  TITLE: ${{ github.event.client_payload.title }}
+  OWNER: "Esri"
+  REPO: "calcite-components"
+
+jobs:
+  existing_issue:
+    name: Find Existing Issue
+    runs-on: ubuntu-latest
+    outputs:
+      github_issue: ${{ steps.find-existing-issue.outputs.result }}
+    steps:
+      - uses: actions/github-script@v6
+        id: find-existing-issue
+        env:
+          NUMBER: "${{env.NUMBER}}"
+          OWNER: "${{env.OWNER}}"
+          REPO: "${{env.REPO}}"
+          TITLE: "${{env.TITLE}}"
+          ACTION: "${{github.event.client_payload.action}}"
+        with:
+          result-encoding: string
+          script: |
+            const { NUMBER, OWNER, REPO, ACTION, TITLE } = process.env;
+            try {
+                const issue = await github.rest.issues.get({
+                    issue_number: NUMBER,
+                    owner: OWNER,
+                    repo: REPO,
+                });
+
+                return issue.status === 200 ? JSON.stringify(issue) : JSON.stringify(null);
+            } catch (error) {
+                
+                const issue = await github.rest.search.issuesAndPullRequests({
+                    q: `"${TITLE}" in:title is:open is:issue`,
+                    sort: 'created',
+                    order: 'asc',
+                    per_page: 1
+                }).then((result) =>  result[0]);
```

Comment:
`result` can't be null. This method either resolves to an array or throws due to an error or invalid query. This may need a try/catch to deal with the latter.